webapp/channels/src/packages/mattermost-redux/src/actions/channels.ts

```diff
@@ -701,6 +702,27 @@ export function updateApproximateViewTime(channelId: string): ActionFuncAsync {
     };
 }
 
+export function unsetActiveChannelOnServer(): ActionFuncAsync {
+    return async (dispatch, getState) => {
+        const state = getState();
+        const currentUser = getCurrentUserId(state);
+        if (!currentUser) {
```

Comment:
Why check if there's a user before doing this? As far as I know, the `TeamController` is mounted when we have a current user, so this check shouldn't be needed

Reply:
I was probably copy pasting something. Will review and remove if it makes sense.